core/js/decanter.js

```diff
@@ -1,3 +1,426 @@
-/**
- * Empty until further notice.
- */
+// if NodeList doesn't support forEach, use Array's forEach()
+NodeList.prototype.forEach = NodeList.prototype.forEach || Array.prototype.forEach;
```

Comment:
This is next up, I will need to pull out my debugger to run through this. Excited to work on some ES6+